package.json

```diff
@@ -66,10 +66,11 @@
     "@vitejs/plugin-vue2": "^1.1.2",
     "@vue/cli-plugin-typescript": "^4.5.11",
     "@vue/cli-service": "^4.5.11",
-    "@vue/test-utils": "^1.3.0",
+    "@vue/test-utils": "^1.3.6",
```

Comment:
Were there issues with running the tests? We recently switched from jest to vitest, so these dependency updates shouldn't be needed.